src/pages/patientView/vafPlot/VAFPlot.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import $ from 'jquery';
+// @ts-ignore
```

Comment:
can you add a comment explaining why

Reply:
Good catch! I was experimenting with tsconfig options when I was trying to expose the whole `shared` directory as is. We don't need to modify this file at this point, I'll revert it.